pkg/cloudprovider/aws/fake/ec2api.go

```diff
@@ -68,6 +68,7 @@ func (e *EC2API) Reset() {
 	e.EC2Behavior = EC2Behavior{
 		CalledWithCreateFleetInput:          set.NewSet(),
 		CalledWithCreateLaunchTemplateInput: set.NewSet(),
+		DescribeSecurityGroupsOutput:        nil,
```

Comment:
You don't need to include this, since by default it's nil. There are other fields that do the same thing.